src/components/TeamMember/ExpandedInfo/ExpandedInfo.styles.tsx

```diff
@@ -0,0 +1,120 @@
+import styled from "styled-components";
+import Image from "next/image";
+import Link from "next/link";
+import { Icon as DefaultIcon } from "@/components/Icon/Icon";
+
+export const ExpandedInfoWrapper = styled.div`
+  display: flex;
+  border-radius: 0.8rem;
```

Comment:
Instead of using rem, its better to use px.

```suggestion
  border-radius: 6px;
```